app/retail/templates/emails/new_kudos.html

```diff
@@ -55,57 +60,128 @@
-  </h3>
-  <div>
-    <img src="{{ token_elem.img_url }}" alt="{{ token_elem.name|humanize_name }}" width="200" style="margin:auto;">
-  </div>
-</div>
+  .kudos-fancy-bg h1 {
+    color: #FFFFFF;
+    letter-spacing: 0;
+    text-align: center;
+    line-height: 59px;
+  }
 
-<div class="centered-contents kudos-box">
-  <div class="kudos">
-    <div class="kudos-contents">
-      {% if kudos_transfer.github_url %}
-        <p>
-        {% if kudos_transfer.from_name %}{{ kudos_transfer.from_name }}{% else %}The sender{% endif %} said it was for your work on <a href="{{ kudos_transfer.github_url }}">{{ kudos_transfer.github_url }}</a>.
-        {% if comments %}
-          {% trans "They had the following comments:" %}
-          <pre>{{ comments }}</pre>
-        {% endif %}
-        </p>
-      {% endif %}
+  .kudos-fancy-bg h3 {
+    color: #FFFFFF;
+    letter-spacing: 0;
+    text-align: center;
+    line-height: 50px;
+  }
+
+  .kudos-fancy-bg p {
+    color: #FFFFFF;
+  }
+
+  .kudos-fancy-bg .kudos-sparkle {
+    position: absolute;
+    background-image: url("{% static 'v2/images/kudos/kudos-sparkle.png' %}");
+    width: 185px;
+    height: 178px;
+    top: 85%;
+    left: 15%;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: 70px;
+  }
+
+  .kudos-fancy-bg .kudos-sparkle-reverse {
+    transform: scaleX(-1) scaleY(-1);
+    left: 60%;
+    top: -10%;
+  }
 
+  .kudos-fancy-bg .kudos-hex-container {
+    position: relative;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: 150px;
+    height: 360px;
+  }
+
+  .kudos-fancy-bg .kudos-hex-content {
+    position: relative;
+    top: 10%;
+    /*top: 19%;*/
+  }
+
+  .kudos-fancy-bg .kudos-hex-content img {
+      position: relative;
+    width: 330px;
+    max-width: 100%;
+  }
+</style>
+<div>
+  <div class="kudos-fancy-bg">
+    <h1>
+      {% if is_new %}
+        {% trans "Awesome!" %}
+      {% else %}
+         🕑 {% trans "Reminder: Kudos Expires Soon" %} 🕝
+      {% endif %}
+    </h1>
+    <h3>
+      You received a <b>{{ token_elem.name|humanize_name }}</b> kudos
+      {% if kudos_transfer.from_name %}
+        from <b>{{ kudos_transfer.from_name }}</b>
+      {% endif %}
+      !
+    </h3>
+    {% if not is_new %} 
+      {{ kudos_transfer.created_on|naturaltime }}
+    {% endif %}
+    <div class="kudos-hex-container">
+      <div class="kudos-sparkle">
+        &nbsp;
+      </div>
+      <div class="kudos-sparkle kudos-sparkle-reverse">
+        &nbsp;
+      </div>
+      <div class="kudos-hex-content">
+        <img src="{{ token_elem.img_url }}" alt="{{ token_elem.name|humanize_name }}">
+      </div>
+    </div>
+    <BR>
+    <p>
       {% if comments_priv %}
-        <p>
-          {% trans "They had the following private comments:" %}
-          <pre>{{ comments_priv }}</pre>
-        </p>
+        {{ comments_priv }}<BR>
```

Comment:
Can we switch to `<br>` to remain consistent with the rest of the codebase?

Reply:
yes we can.